umbraco/Umbraco-CMS
# Replaced deprecated navigator.platform with navigator.userAgent for platform detection.

This fixes #13156 

### Description
Browser OS information uses the deprecated method navigator.platform. So I replaced it with `navigator.userAgent`, exactly the same as how it is used in umbraco v15.

**Note:** Though the issue is reported for v10 and v11. It is present in v13 LTS as well.

### How to test
1. Log into the Umbraco Backoffice.
2. Click on the Help icon in the top right of the screen.
3. View the System Information from the sidebar that appears.
4. Check the Browser (user agent) string

![image](https://github.com/user-attachments/assets/f88e1003-c468-4bda-97c6-03ef5c5202d7)
